sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -105,27 +106,19 @@ def main(unused_argv):
   else:
     fn_log_handler = None
 
-  # Start status HTTP server thread.
-  thread = threading.Thread(
-      name='status_http_server', target=StatusServer().start)
-  thread.daemon = True
-  thread.setName('status-server-demon')
-  thread.start()
-
-  if 'PIPELINE_OPTIONS' in os.environ:
-    sdk_pipeline_options = _parse_pipeline_options(
-        os.environ['PIPELINE_OPTIONS'])
-  else:
-    sdk_pipeline_options = PipelineOptions.from_dictionary({})
+  pipeline_options_dict = _load_pipeline_options(
+      environment.get('PIPELINE_OPTIONS'))
+  RuntimeValueProvider.set_runtime_options(pipeline_options_dict)
```

Comment:
Better to add a comment here to describe its purpose.

Reply:
Done.